src/lbaf/Applications/conf.yaml

```diff
@@ -72,7 +72,7 @@ algorithm:
 
 # Specify output
 #logging_level: debug
-show_traceback: False
+show_traceback: True
```

Comment:
Please eliminate this option, we don't want this any longer.  We will always have the 2 following modes:
1- pre-defined error condition: `logger.error` + exception --> termination without stack trace
2- undefined error: runtime error + stack trace

Therefore we no longer need this `show_traceback` error

Reply:
@ppebay  For that I created an [issue](https://github.com/DARMA-tasking/LB-analysis-framework/issues/272), to do when I will have some capacity. Rest is done.

Reply:
Okay got it. So we leave it like that for now.